### DISCUSSION TITLE
서브모듈 레포의 접근성에 대해

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/01-doo-re-back#72] [논의 사항]
현재 구현된 서브모듈 레포는 BDD organization 소속으로 생성이 되어 있습니다.
private 레포이기 때문에 BDD 외부 회원들에겐 노출되지 않겠지만, BDD organization에 소속된 회원들은 접근할 수 있어서 보안상으로 위험하지 않을까 걱정이 됩니다.
그래서 서브모듈 레포를 누군가의 개인 레포로 생성하는 것은 어떨지 건의드려봅니다.

**Comment by lcqff** [comment]
현경이 개인 레포에 만드는 걸로
